cohort-cli/src/main/java/com/ibm/cohort/cli/CohortCLI.java

```diff
@@ -79,9 +79,15 @@
 		private LibraryFormat sourceFormat = DEFAULT_SOURCE_FORMAT;
 		
 		@Parameter(names = { "-i",
-		"--model-info" }, description = "Model info file used when translating CQL", required = false)
+				"--model-info" }, description = "Model info file used when translating CQL", required = false)
 		private File modelInfoFile;
-
+		
+		@Parameter(names = { "--expand-value-sets" }, arity = 1, description = "By default, ValueSet resources used in CQL are expanded by the terminology server. If the data server supports terminology and the token :in modifier, setting this flag to false will enable use of that functionality which should improve CQL engine throughput.", required = false )
+		private boolean expandValueSets = DEFAULT_EXPAND_VALUE_SETS;
```

Comment:
Would it make more sense to name this `--disable-value-set-expansion` or something?  I try to steer away from having users put `true` or `false` after boolean parameters.  This is borderline pedantic though so if no one else has an issue, then I'm good.